src/workerd/api/crypto/impl.h

```diff
@@ -362,6 +362,13 @@ class ZeroOnFree {
 // be thrown. Otherwise the method will return normally.
 void checkPbkdfLimits(jsg::Lock& js, size_t iterations);
 
+// Either succeeds with exactly |length| bytes of cryptographically
+// strong pseudo-random data, or fails. This function may block.
+// Don't assume anything about the contents of |buffer| on error.
+// As a special case, |length == 0| can be used to check if the CSPRNG
```

Comment:
Comment can be simplified as we only support the simplified check now. Also bufer => buffer

Reply:
can you suggest a specific simplification